src/bench/readblock.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <bench/bench.h>
+#include <bench/data.h>
+
+#include <chainparams.h>
+#include <consensus/validation.h>
+#include <node/blockstorage.h>
+#include <streams.h>
+#include <util/system.h>
+#include <validation.h>
+
+using node::BLOCK_SERIALIZATION_HEADER_SIZE;
+using node::OpenBlockFile;
+using node::ReadBlockFromDisk;
+using node::ReadRawBlockFromDisk;
+
+static void SetupReadBlockTest()
+{
+    // Create params and set blocksdir to /tmp folder
+    SelectParams(CBaseChainParams::MAIN);
+    gArgs.ForceSetArg("-blocksdir", "/tmp");
```

Comment:
I'm wondering can't we do something smarter here to make it work also on Windows?

Reply:
Good suggestion. Updated to use `fs::temp_directory_path()`.